snapcraft/internal/project_loader/_config.py

```diff
@@ -88,7 +88,7 @@ def _remote_parts(self):
 
     def __init__(self, project_options=None):
         if project_options is None:
-            project_options = snapcraft.ProjectOptions()
+            project_options = snapcraft.Project()
```

Comment:
This should probably be called `project` now, not `project_options`.

Reply:
For the record: clarified in an offline conversation that this would be confusing without changing all the classes used by `Config` and porting all the consumers would make this PR too big so I'm going to do that in a follow-up branch.